docs/user/metrics/github.csv

```diff
@@ -8,5 +8,6 @@ Library,Contributors,Forks,Watchers,Stars,Issues,PRs,Releases,Commits,Core SLOCs
 `TerraTorch`_,16,24,12,171,78,185,8,606,"14,933","2,077",44%,Apache-2.0
 `SITS`_,14,78,28,483,654,590,44,"6,244","24,284","8,697",94%,GPL-2.0
 `scikit-eo`_,7,20,9,192,24,13,17,510,"1,617","170",37%,Apache-2.0
+`geo-bench`_,6,8,11,108,16,7,3,"1211",,,Apache-2.0
```

Comment:
* **Commits**: 1211 -> 1,211 (American style 😉)
* **Core SLOCs**: 2,349
* **Test SLOCs**: 277
* **Test Coverage**: If you have all the deps installed, should be easy to calculate, but let me know if you need help.